src/modules/form/components/viewable/DynamicViewField.tsx

```diff
@@ -103,9 +106,27 @@ const DynamicViewField: React.FC<DynamicViewFieldProps> = ({
         />
       );
     case ItemType.TimeOfDay:
+      return (
+        <TextContent
+          {...commonProps}
+          renderValue={(val) => formatTimeOfDay(val)}
```

Comment:
Read-only display for time of day

<img width="360" alt="Screenshot 2024-02-22 at 3 44 20 PM" src="https://github.com/greenriver/hmis-frontend/assets/5333982/8263f449-17ac-4a4b-bb31-5477a69f6f5b">